tf_files/modules/cdis-aws-common-logging/logging.tf

```diff
@@ -189,77 +185,73 @@ data "aws_iam_policy_document" "firehose_policy_document" {
 }
 
 resource "aws_iam_role_policy" "firehose_policy" {
-  name   = "${var.child_name}_firehose_policy"
+  name   = "${var.common_name}_firehose_policy"
   policy = "${data.aws_iam_policy_document.firehose_policy_document.json}"
   role   = "${aws_iam_role.firehose_role.id}"
 }
 
 # Need these guys because the firehose resource is not that smart to create it if it doesn't exist
 
-resource "aws_cloudwatch_log_group" "csoc_log_group" {
-  name = "${var.child_name}"
+resource "aws_cloudwatch_log_group" "csoc_common_log_group" {
+  name = "${var.common_name}"
```

Comment:
So we have a log-group per commons in the CSOC?  What logs go there - just the `adminvm`?  
It looks like the `firehost_to_s3` is associated with this log_group?  Does the firehose drop logs here, then?
Maybe just add a comment explaining what's going on for people like me ... :-p

Reply:
Commenting the code, but basically firehoses would put anything there if it errors for whatever reason and cant' reach the destination.

Have not happened yet, but that should help understanding if firehoses fail